### ISSUE TITLE
Enhance a11y for `SegmentedControl`

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#5412]
name: SegmentedControl

### Description
8 accessibility violations were found in the `SegmentedControl`.

#### [Components / SegmentedControl / ReadOnly](https://yamada-ui.github.io/yamada-ui/?path=/story/components-segmentedcontrol--read-only)

Received:
Elements must only use supported ARIA attributes (aria-allowed-attr)

Fix all of the following:
  ARIA attribute is not allowed: aria-readonly="true"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/aria-allowed-attr?application=axeAPI

#### [Components / SegmentedControl / ReadOnly](https://yamada-ui.github.io/yamada-ui/?path=/story/components-segmentedcontrol--read-only)

Received:
Elements must only use supported ARIA attributes (aria-allowed-attr)

Fix all of the following:
  ARIA attribute is not allowed: aria-readonly="true"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/aria-allowed-attr?application=axeAPI

#### [Components / SegmentedControl / ReadOnly](https://yamada-ui.github.io/yamada-ui/?path=/story/components-segmentedcontrol--read-only)

Received:
Elements must only use supported ARIA attributes (aria-allowed-attr)

Fix all of the following:
  ARIA attribute is not allowed: aria-readonly="true"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/aria-allowed-attr?application=axeAPI

#### [Components / SegmentedControl / ReadOnly](https://yamada-ui.github.io/yamada-ui/?path=/story/components-segmentedcontrol--read-only)

Received:
Elements must only use supported ARIA attributes (aria-allowed-attr)

Fix all of the following:
  ARIA attribute is not allowed: aria-readonly="true"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/aria-allowed-attr?application=axeAPI

#### [Components / SegmentedControl / ReadOnly](https://yamada-ui.github.io/yamada-ui/?path=/story/components-segmentedcontrol--read-only)

Received:
Elements must only use supported ARIA attributes (aria-allowed-attr)

Fix all of the following:
  ARIA attribute is not allowed: aria-readonly="true"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/aria-allowed-attr?application=axeAPI

#### [Components / SegmentedControl / ReadOnly](https://yamada-ui.github.io/yamada-ui/?path=/story/components-segmentedcontrol--read-only)

Received:
Elements must only use supported ARIA attributes (aria-allowed-attr)

Fix all of the following:
  ARIA attribute is not allowed: aria-readonly="true"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/aria-allowed-attr?application=axeAPI

#### [Components / SegmentedControl / ReadOnly](https://yamada-ui.github.io/yamada-ui/?path=/story/components-segmentedcontrol--read-only)

Received:
Elements must only use supported ARIA attributes (aria-allowed-attr)

Fix all of the following:
  ARIA attribute is not allowed: aria-readonly="true"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/aria-allowed-attr?application=axeAPI

#### [Components / SegmentedControl / ReadOnly](https://yamada-ui.github.io/yamada-ui/?path=/story/components-segmentedcontrol--read-only)

Received:
Elements must only use supported ARIA attributes (aria-allowed-attr)

Fix all of the following:
  ARIA attribute is not allowed: aria-readonly="true"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/aria-allowed-attr?application=axeAPI

### Additional Information
While it's ideal to cover all tests, if there are too many lines to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.
